content/en/platform/corda/1.4/cenm/deployment-kubernetes.md

```diff
@@ -608,15 +607,15 @@ The Docker images used for the Kubernetes deployment are listed below for refere
 
 {{< table >}}
 
-| Service           | Image name                                          |
-|-------------------|-----------------------------------------------------|
-| Identity Manager  | `corda/enterprise-identitymanager:1.4-zulu-openjdk8u242 `     |
+| Service           | Image name                                                |
```

Comment:
List of images for the 1.4.4 release:
```
corda/enterprise-identitymanager:1.4.4-zulu-openjdk8u242
corda/enterprise-networkmap:1.4.4-zulu-openjdk8u242
corda/enterprise-pkitool:1.4.4-zulu-openjdk8u242
corda/enterprise-signer:1.4.4-zulu-openjdk8u242
corda/enterprise-zone:1.4.4-zulu-openjdk8u242

corda/enterprise-auth:1.0.3-zulu-openjdk8u242
corda/enterprise-gateway:1.1.2-zulu-openjdk8u242

corda/enterprise-notary:4.4.11-zulu-openjdk8u242
```

Reply:
In all `deployment-kubernetes-<service>.md` files under `1.4/cenm/`, the image tag does not match the latest release and is `1.4` instead. Do we want to update those entries with the list above?

Reply:
Applied to CE4.6 and 1.4.